pkg/kotsadm/push_images.go

```diff
@@ -272,9 +272,10 @@ func TagAndPushAppImages(imagesDir string, options types.PushImagesOptions) ([]k
 
 		reportWriter := options.ProgressWriter
 		if options.LogForUI {
-			reportWriter := reportWriterWithProgress(imageFiles, options.ProgressWriter)
-			defer reportWriter.Write([]byte(fmt.Sprintf("+status.flush:\n")))
-			defer reportWriter.Close()
+			wc := reportWriterWithProgress(imageFiles, options.ProgressWriter)
+			reportWriter = wc.(io.Writer)
```

Comment:
It's not clear that anything at all happened here. Would you mind explaining what this changed?

Reply:
@MikaelSmith A new `reportWriter` variable was created inside this if statement block instead of reassigning the existing `reportWriter` which is used to write the "pushing images" progress messages.

Reply:
Ah I see it now. Thanks.